examples/meta-counter/src/service.rs

```diff
@@ -25,13 +26,16 @@ impl Service for MetaCounterService {
     type Storage = SimpleStateStorage<Self>;
     type State = MetaCounter;
 
-    async fn new(state: Self::State, _runtime: ServiceRuntime) -> Result<Self, Self::Error> {
-        Ok(MetaCounterService { _state: state })
+    async fn new(state: Self::State, runtime: ServiceRuntime<Self>) -> Result<Self, Self::Error> {
+        Ok(MetaCounterService {
+            _state: state,
```

Comment:
Do we even need that field, or should we drop the state view right away?

Reply:
Good point, thanks!